### DISCUSSION TITLE
Should `interactCoupled` methods get added to docs?

### DISCUSSION THREAD

**Discussion by albeanth** [terrapower/armi#1577] [General]
https://github.com/terrapower/armi/blob/4d3aafde25625ab1424f77ba61af39e3ad11b6c5/doc/developer/guide.rst?plain=1#L199

discusses the different interface interaction hooks but `interactCoupled` is left out. Should this get added?

**Comment by keckler** [comment]
Yes

**Reply by john-science** [reply]
Agreed. Open a ticket for me.  Or, you know, a PR.  But one of the two, so I don't lose this. :)

**Reply by john-science** [reply]
PR is up: https://github.com/terrapower/armi/pull/1581

**Comment by john-science** [comment]
PR is up: https://github.com/terrapower/armi/pull/1581
